Kitodo/src/main/java/org/kitodo/production/forms/prozesskopie/TitleRecordLinkTab.java

```diff
@@ -0,0 +1,332 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.forms.prozesskopie;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Locale.LanguageRange;
+import java.util.Objects;
+
+import javax.faces.model.SelectItem;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kitodo.api.dataeditor.rulesetmanagement.RulesetManagementInterface;
+import org.kitodo.api.dataeditor.rulesetmanagement.StructuralElementViewInterface;
+import org.kitodo.api.dataformat.IncludedStructuralElement;
+import org.kitodo.api.dataformat.Workpiece;
+import org.kitodo.data.database.beans.Process;
+import org.kitodo.data.database.exceptions.DAOException;
+import org.kitodo.data.exceptions.DataException;
+import org.kitodo.production.dto.ProcessDTO;
+import org.kitodo.production.forms.ProzesskopieForm;
+import org.kitodo.production.helper.Helper;
+import org.kitodo.production.metadata.MetadataEditor;
+import org.kitodo.production.services.ServiceManager;
+import org.kitodo.production.services.data.ProcessService;
+import org.primefaces.model.DefaultTreeNode;
+import org.primefaces.model.TreeNode;
+
+/**
+ * Backing bean for the ‘title record link’ tab of the process creation dialog.
+ */
+public class TitleRecordLinkTab {
+    private static final Logger logger = LogManager.getLogger(TitleRecordLinkTab.class);
+
+    /**
+     * Maximum number of search hits to show.
+     */
+    private static final int MAXIMUM_NUMBER_OF_HITS = 10;
+
+    /**
+     * Process creation dialog to which this tab belongs.
+     */
+    private final ProzesskopieForm prozesskopieForm;
+
+    /**
+     * List of search hits to show.
+     */
+    private List<HitLink> hitLinks = Collections.emptyList();
```

Comment:
Isn't the result a list of processes?

Reply:
The list of selectable hits was a list of objects whose class implemented an action when clicked. This was replaced as requested by a selectOneMenu.